tests/test_pickle.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 import numpy as np
 from sympy import Symbol
-import pickle
+import cloudpickle as pickle
```

Comment:
As above, import both and make all tests for each

Reply:
ah maybe i see what you mean now... are you saying something like

```
import pickle as pickle0
import cloudpickle as pickle1

@pytest.parametrize('pickle', [pickle0, pickle1])
def test_...(...):
    ...
```

?

isn't it overkill?

Reply:
Yes what I mean. Since there seems to be breaking difference between those two pickling implementation I think it would be safer to have it yeah. THese are very cheap tests so it shouldn't be an issue

Reply:
done. Please take a look, I've restructured a few things